core/modules/memman/MemFile.h

```diff
@@ -50,7 +50,7 @@ class MemFile {
     struct MLResult {
         uint64_t bLocked;
         int      retc;
-        MLResult() {}
+        MLResult() : bLocked{false}, retc(0) {}
```

Comment:
Ah, false is the wrong type here. Yeah it will work but it's inconsistent, use 0 instead.